packaging/scripts/postinstall.sh

```diff
@@ -6,6 +6,67 @@ ETC_SERVICE="/etc/systemd/system/evcc.service"
 USR_LOCAL_BIN="/usr/local/bin/evcc"
 RESTART_FLAG_FILE="/tmp/.restartEvccOnUpgrade"
 
+
+# Call /usr/bin/evcc checkconfig and capture the output
+# if exit code is 0, then remove /tmp/oldevcc
+# if exit code is not 0, then fail installation with error message and copy /tmp/oldevcc back to /etc/evcc
+# if /tmp/oldevcc does not exist, then do nothing
+failInstallation=0
+INTERACTIVE=0
+# is shell script interactive?
+if [ -t 0 ]; then
+  INTERACTIVE=1
+else
+  INTERACTIVE=0
+fi
+
+if [ -f /tmp/oldevcc ] && [ $INTERACTIVE -eq 1 ]; then
+    checkConfigOutput=$(/usr/bin/evcc checkconfig 2>&1 || true)
+	oldevccversion=$(/tmp/oldevcc -v)
+	newevccversion=$(/usr/bin/evcc -v)
+    if echo "$checkConfigOutput" | grep -q "config valid"; then
+ 			
+			if [ "$oldevccversion" = "$newevccversion" ]; then
+				echo "--------------------------------------------------------------------------------"
+				echo "Old evcc version detected. To apply the new version, please reinstall evcc. (e.g. apt-get reinstall evcc)"
+				echo "--------------------------------------------------------------------------------"
+			else 
+			rm -rf /tmp/oldevcc
+			fi
+	else
+			if [ "$oldevccversion" = "$newevccversion" ]; then
+				echo "--------------------------------------------------------------------------------"
+				echo "Old evcc version detected. To apply the new version, please reinstall evcc. (e.g. apt-get reinstall evcc)"
+				echo "--------------------------------------------------------------------------------"
+			else 
+				echo "--------------------------------------------------------------------------------"
+				echo "ERROR: your evcc configuration is not compatible with the new version. Please consider reading the release notes: https://github.com/evcc-io/evcc/releases"
+				echo "checkconfig Output:" 
+				echo "$checkConfigOutput"
+				echo "--------------------------------------------------------------------------------"
+        
+		while true; do
```

Comment:
Either this while-block is incorrectly indented, or it is accidentally added to the else branch above. (Or Github UI is messing up my view...)

Reply:
The indentation issue has been resolved. However, I can confirm that the placement of the loop is correct.